packages/provider/wrapper.js

```diff
@@ -1,6 +1,38 @@
 const debug = require("debug")("provider:wrapper"); // eslint-disable-line no-unused-vars
 const ProviderError = require("./error");
 
+function make1193ProviderRequest(provider) {
+  let requestId = 0;
```

Comment:
In my opinion, since this will only matter when we use WebSockets instead of HTTP, we can keep things simple for now having only one `requestId` counter. And come back to it once we are done releasing the `truffle call` feature. I think I need some more clarity on this.

Reply:
If that's the case, a note in the code wrt `requestId` would be prudent.

Reply:
It makes sense to add a note in there to remind of any future work related to the `requestId ` to be done. I will do that, thanks @cds-amal .